src/Common/tests/System/Net/Http/HuffmanEncoder.cs

```diff
@@ -281,32 +280,52 @@ public static int GetEncodedLength(ReadOnlySpan<byte> src)
             return (bits + 7) / 8;
```

Comment:
When you are here...

```suggestion
             return (bits + 7) >> 3;
```
Will emit `lea` and `sar` instead of the integer division.
RyuJit will perform this optimization only for unsigneds.

Reply:
This is test code. Clarity is preferred over perf.